files/usr/lib/jeos-firstboot

```diff
@@ -63,13 +63,13 @@ fi
 modules=()
 
 call_module_hook() {
-    hook="$1"
+    local hook="$1"
     for module in "${modules[@]}"; do
         hook_function="${module}_${hook}"
-        [ $(type -f "${hook_function}") == "function" ] || continue
-        "${hook_function}" && true # To not trigger errexit
+        [ $(type -f "${hook_function}") = "function" ] || continue
+        "${hook_function}" || true # To not trigger errexit
```

Comment:
That way `ret` is always 0.

Reply:
what is the function supposed to do in the error case anyways? you could of course do
   "$hook_function" || return $?

but then the script would die later. So probably printing some error and continuing is better

Reply:
I'll vote for error printing and then in distant future when there is more time and knownledge script would handle errors somehow..